postgresql/plan.sh

```diff
@@ -30,9 +30,10 @@ do_build() {
 	# executable, the contents of the environment variable LD_RUN_PATH
 	# will be used if it is defined"
 	./configure --disable-rpath \
-              --with-openssl \
-              --prefix=${pkg_prefix} \
-              --sysconfdir=${pkg_svc_config_path} \
-              --localstatedir=${pkg_svc_var_path}
+                    --with-openssl \
+                    --enable-thread-safety \
```

Comment:
Default is enabled. Typically affected the `libpq` library, which has been thread-safe from ~9.0

Can you provide some more context where this flag comes into play?